src/redisai.h

```diff
@@ -264,6 +268,7 @@ static int RedisAI_Initialize(RedisModuleCtx *ctx) {
     REDISAI_MODULE_INIT_FUNCTION(ctx, TensorCreate);
     REDISAI_MODULE_INIT_FUNCTION(ctx, TensorCreateByConcatenatingTensors);
     REDISAI_MODULE_INIT_FUNCTION(ctx, TensorCreateBySlicingTensor);
+    REDISAI_MODULE_INIT_FUNCTION(ctx, TensorCreateFromDLTensor);
```

Comment:
why this is on redisai.h?

Reply:
I added this to the LLAPI when I saw that I need it, but I guess that we don't need it there if we have `backends_api.h`...